NeoSharp.TransactionManager/ITransactionManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using NeoSharp.Core.Blockchain;
+using NeoSharp.Core.Models;
+using NeoSharp.Core.Types;
+using NeoSharp.Wallet;
+
+namespace NeoSharp.TransactionManager
+{
+    public interface ITransactionManager : INEP5TransactionManager
+    {
+
+
+        Transaction BuildUnsignedTransaction(
+            IWalletAccount from,
+            List<TransactionAttribute> attributes,
+            List<CoinReference> inputs,
+            List<TransactionOutput> outputs,
+            double fee = 0);
+
+        Transaction BuildUnsignedTransaction(
+            IWalletAccount from,
+            List<TransactionAttribute> attributes,
+            List<TransactionOutput> outputs,
+            double fee = 0);
+
+
```

Comment:
Please tabule right the methods

Reply:
Hi @shargon, are you following a specific code style? How this line should be indented?